### PR TITLE
Assembly AI

### PR DESCRIPTION
**Pull request by dhruv1220**

### CODE REVIEW

File: AssemblyAI/__pycache__/utils.cpython-310.pyc
Comment (SudhanshuBlaze) [inline AssemblyAI/__pycache__/utils.cpython-310.pyc:1]: Remove pycache file or add it in gitignore

File: .DS_Store
Comment (SudhanshuBlaze) [inline .DS_Store:1]: Add it to gitignore as well

**Comment by dhruv1220** [comment]
@SudhanshuBlaze added the gitignore file as requested. Let me know if it's okay now

**Review by SudhanshuBlaze — APPROVED** [review]
LGTM!

**Comment by dhruv1220** [comment]
@SudhanshuBlaze Github is saying that merging for this branch is blocked as all commits need to be signed. How do I do that?

**Comment by SudhanshuBlaze** [comment]
You need to setup your git to sign the commits automatically, follow the below article :

https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Comment by dhruv1220** [comment]
@SudhanshuBlaze lol so I was able to sign my commits but my previous commits still remain unsigned which isn't allowing merging the PR. Apparently, I'll have to sign each commit individually. If it's super important, I'll do that but if we can work around it, that would be easier for me xD @afiDPS tagging you as well to know your opinion

**Comment by dhruv1220** [comment]
> I think it's possible to sign previous commits as well. Or maybe Afi can merge this PR.

Yes yes you're right it is possible just that it's a tedious process that's all

**Pull request merged**